docs/DeveloperGuide.md

```diff
@@ -232,11 +232,15 @@ Finer details than what is shown in the section below can be seen under the impl
 
 ![Structure of the Storage Component](images/StorageClassDiagram.png)
 
+**Diagram Notes**:
+* `Storage` implements `CommandHistoryStorage`, `UserPrefsStorage` and `AddressBookStorage`. However, this interface inheritance is omitted for brevity.
```

Comment:
Hmm, should "implements" be "inherits from"? Technically, interfaces cannot implement anything.

Reply:
Updated!